src/custom-widget.ts

```diff
@@ -33,20 +21,25 @@ window.TagoIO = {} as ITagoIO;
  */
 window.TagoIO.autoFill = true;
 
-let funcRealtime: ICallbackRealtime;
-let funcStart: ICallbackStart;
-let funcError: ICallbackError;
-// array with widget variables to enable autoFill
-let widgetVariables: Array<ITagoVariables>;
-const pool: Array<(data: IData | null, error?: IError) => void> = [];
+let funcRealtime: TRealtimeCallback;
+let funcStart: TStartCallback;
+let funcError: TErrorCallback;
+let funcSyncUserInfo: TUserInformationCallback;
+let widgetVariables: TWidgetVariable[];
+
+const pool: Array<(data: TData | null, error?: TError) => void> = [];
 
 /**
  * eventListener function that receives messages sent by the parent component
  * @param event event coming from the parent component
  */
-const receiveMessage = (event: IEvent): void => {
+const receiveMessage = (event: TEvent): void => {
   const { data } = event;
   if (data) {
+    if (data.userInformation) {
+      funcSyncUserInfo(data.userInformation);
```

Comment:
Uses the callback to sync user information (token, language, and RUN URL) with the Custom Widget code.

Main use for this is being able to pass the information required for using `Dictionary` from the SDK inside of the Custom Widget code.